src/ceph-disk/ceph_disk/main.py

```diff
@@ -665,6 +665,19 @@ def get_dev_size(dev, size='megabytes'):
         os.close(fd)
 
 
+def is_diskdevice
```

Comment:
@dachary 
I also see things happening on files. That is certainly not ISBLK.
Also there are in Filestore directories on which to build the Ceph stuff.
Again not ISBLK I think?